### PR TITLE
Fix yaml formatting

### PR DESCRIPTION
**Pull request by x10an14**

### CODE REVIEW

**Comment by sondr3** [comment]
Pulled and checked locally, LGTM :shipit:

**Review by sondr3 — APPROVED** [review]
Pulled and checked locally, LGTM :shipit:

File: src/main/kotlin/io/nais/NaisApplication.kt [hunk 1]
```diff
@@ -71,8 +70,20 @@ data class Prometheus(
...
+   val cpu: String,
+   val memory: String,
+)
+
+@Serializable
+data class Limits(
    val cpu: String,
-   val memory: String
+   val memory: String,
 )
```
Comment (jksolbakken) [inline src/main/kotlin/io/nais/NaisApplication.kt:88]: No need for two different classes with the same members

**Review by jksolbakken — APPROVED** [review]
See the comment, otherwise 👍

File: src/main/kotlin/io/nais/NaisApplication.kt (hunk 1 above)
Comment (x10an14) [reply src/main/kotlin/io/nais/NaisApplication.kt:82]: Fix'd! =)

**Pull request merged**